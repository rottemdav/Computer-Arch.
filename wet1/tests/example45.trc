1 5 21 2 global_history local_tables not_using_share
0x9a4 T 0x7fcc34
0x48a0 N 0xa822f4
0x7da63d18 N 0x9801274
0xd54 T 0x7e84
0x10c T 0xaf37b090
0x94c28 N 0xda802e0
0x58bc60 T 0x7f2fed88
0xd54 N 0x7e84
0x35cb00 N 0xc6e18
0x13d2d294 T 0xe4811dd0
0x8ec39674 N 0xf96fae4
0x94c28 N 0xda802e0
0x35cb00 T 0xc6e18