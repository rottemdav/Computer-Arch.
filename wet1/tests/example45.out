0x9a4 N 0x9a8
0x48a0 N 0x48a4
0x7da63d18 N 0x7da63d1c
0xd54 N 0xd58
0x10c N 0x110
0x94c28 N 0x94c2c
0x58bc60 N 0x58bc64
0xd54 N 0xd58
0x35cb00 N 0x35cb04
0x13d2d294 N 0x13d2d298
0x8ec39674 N 0x8ec39678
0x94c28 N 0x94c2c
0x35cb00 N 0x35cb04
flush_num: 6, br_num: 13, size: 121b
